src/main/java/com/pinterest/secor/tools/TestLogMessageProducer.java

```diff
@@ -55,17 +56,15 @@ public TestLogMessageProducer(String topic, int numMessages, String type,
     public void run() {
         Properties properties = new Properties();
         if (mMetadataBrokerList == null || mMetadataBrokerList.isEmpty()) {
-            properties.put("metadata.broker.list", "localhost:9092");
+            properties.put("bootstrap.servers", "localhost:9092");
```

Comment:
Will this property name change break the tests under older kafka version?

Reply:
It doesn't break the tests because I migrated this class to use the new producer. I was able to do this migration because the old consumer dependency also includes the new consumer

Reply:
sounds good